docs/assets/js/search.js

```diff
@@ -112,4 +134,6 @@ if ( searchTerm ) {
 
   // Display the results of the search.
   displaySearchResults( results, searchStore );
+} else {
```

Comment:
Kill this `else` condition and in `search-results.html` add something like `<li>Enter search terms above.</li>` in the unordered list. It'll be removed if search results are found.

Reply:
Won't that show `<li>Enter search terms above.</li>` to the user while the search is in progress potentially if it ends up taking seconds to execute like 18f's does? Also, if there are no search results I was putting in the search term, which is why it was written via JS. Do you think that's too much?

<img width="1342" alt="Screen Shot 2020-05-07 at 4 36 28 PM" src="https://user-images.githubusercontent.com/704760/81342278-f8a50780-9080-11ea-8b20-7293c8182c9c.png">

Reply:
Aw yeah, you're right. Disregard my comments, it works great how it is.